.github/workflows/tutorials.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Install pip dependencies
         if: steps.cache.outputs.cache-hit != 'true'
         run: |
-          pip install -r requirements/required.txt -r requirements/docs.txt -r requirements/tests.txt planetary_computer pystac .
+          pip install -r requirements/required.txt -r requirements/docs.txt -r requirements/tests.txt tensorboard planetary_computer pystac .
```

Comment:
tensorboard is used by the trainers notebook, if we install it here it should get cached and not take up actual test time

Reply:
Can you sort these alphabetically? Also needs to be added to `.github/workflows/release.yaml`. 

Honestly, I wonder if we shouldn't add all 3 deps to the docs or tests extra. I would like to be able to pin these and test specific versions like we do with all other deps.

I kind of want to get rid of planetary_computer/pystac as there are fine tutorials for how to use these and torchgeo has never focused on raw data downloads. But I think things like tensorboard integration are more important to demonstrate, as they directly relate to how to use our trainers.

Can we separate this out into its own PR since it's unrelated to this new tutorial?